src/patterns-hmrc-govuk-team/technical-guidance/index.njk

```diff
@@ -11,11 +11,9 @@ layout: twoColumnPage.njk
 Use this pattern where the majority of the users are tax agents or professionals who need the really complex guidance to deal with their clients affairs.
 </p>
 
-<p class="govuk-body">
-Please send a request to Editorial Board before you start using this pattern, with an explanation about why you think this pattern is right for the content you’re designing.
-</p>
+<p class="govuk-body">You must discuss with a Senior Content Designer if you want to use this pattern for any other reason.</p>
 
-<h2 class="govuk-heading-l">How it works</h2>
+<h2 class="govuk-heading-l">How to design this pattern</h2>
 
 <h3 class="govuk-heading-m">Structure</h3>
 
```

Comment:
Again, can't comment on the actual line as it hasn't been modified but should the `<ol>` below be a `<ul>`?

Reply:
This is going to be reworked soon, so think it's fine as it is for the moment.